tiledb/sm/filter/bitshuffle_filter.cc

```diff
@@ -124,8 +125,8 @@ Status BitshuffleFilter::compute_parts(
 }
 
 Status BitshuffleFilter::shuffle_part(
-    const ConstBuffer* part, Buffer* output) const {
-  auto tile_type = pipeline_->current_tile()->type();
+    const Tile& tile, const ConstBuffer* part, Buffer* output) const {
```

Comment:
minor and matter of taste: in those internal methods of filters such as this one, we could be actually passing what they need, which is the tile.type() instead of the whole Tile.
No problem if you don't want to bother.